### ISSUE TITLE
 Keyboard focus is not visible on header section.

### ISSUE THREAD

**Issue by msftedad** [Hacker0x01/react-datepicker#3771]
**Describe the bug**
 On navigating using Bluetooth Keyboard focus is not visible on header section controls.

**To Reproduce**
1. Launch the Authenticator app.
2. Press the Hamburger button.
3. Select option "Settings".
4. Verify the issue.

**Actual behavior**
While navigating using a Bluetooth keyboard, the keyboard focus is not visible on the header section controls like "Back button and settings text" on the settings screen.

**Expected behavior**
While navigating using a Bluetooth keyboard, the keyboard focus should be visible on the header section like the "Back button and settings text" on the settings screen.

**Screenshots**
https://user-images.githubusercontent.com/93735775/191812996-205fc930-994f-4895-bb17-639d4476ff51.mp4

**Desktop (please complete the following information):**
Not used

**Smartphone (please complete the following information):**
Device: [Samsung S22]
OS: [Android 12]
Application: Authenticator version 6.2204.2416

**Comment by msftedad** [comment]
Hi Team,Any update on this issue?

**Comment by msftedad** [comment]
please don't close this issue as it is still repro'ing.Thanks
